web-nfc/index.js

```diff
@@ -13,8 +13,32 @@ scanButton.addEventListener("click", async () => {
     ndef.addEventListener("reading", ({ message, serialNumber }) => {
       log(`> Serial Number: ${serialNumber}`);
       log(`> Records: (${message.records.length})`);
-    });
-  } catch (error) {
+      
+      if(message.records.length>0){
+      for (const record of message.records) {
+          log("Record type:  " + record.recordType);
```

Comment:
Please use ``log(`Record type: ${record.recordType}`);`` there and below

Reply:
Will follow the proposal. thanks